frontend/src/components/ScheduledEventsTableRow/ScheduledEventsTableRow.tsx

```diff
@@ -34,6 +84,11 @@ export const ScheduledEventsTableRow: FunctionComponent<ScheduledEventsTableRowP
           ? `Every ${props.interval} ${props.interval_type}`
           : 'None'}
       </td>
+      <td>
+        <Button variant="outline-danger" size="sm" onClick={handleClick} data-id={props.id}>
```

Comment:
Add a class `disabled` for when `isLoading` is true ... that way it won't be clicked again.

Reply:
Updated